cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AbstractModel.java

```diff
@@ -109,7 +111,8 @@
 
     public static final String STRIMZI_CLUSTER_OPERATOR_NAME = "strimzi-cluster-operator";
 
-    protected static final Logger log = LogManager.getLogger(AbstractModel.class.getName());
+    protected static final Logger LOGGER = LogManager.getLogger(AbstractModel.class.getName());
+    protected static final ReconciliationLogger RECONCILIATION_LOGGER = ReconciliationLogger.create(AbstractModel.class.getName());
```

Comment:
Just a thought ... would it be better to for example use only the `ReconciliationLogger` and inside it check whether reconciliation is `null` to handle things differently depending on it?